eslint.config.js

```diff
@@ -22,13 +22,13 @@ export default [
       '**/coverage/**/*',
       'packages/ai-api/src/client/**/*',
       'packages/foundation-models/src/azure-openai/client/inference/schema/on-your-data-system-assigned-managed-identity-authentication-options.ts',
+      '**/*.zod.ts'
```

Comment:
[q] Is this rule really needed? Do we commit any files with this extension now?

Reply:
Done